functions/snippets/src/main/java/PubSubMessage.java

```diff
@@ -0,0 +1,26 @@
+/*
```

Comment:
Why isn't this apart of the FF?

Reply:
I don't think the FF should have Google-specific classes like this. However there's a good case to be made for providing another artifact that defines them, rather than having everyone reinvent them.